include/roboteam_ai/stp/new_plays/FreeKickThem.h

```diff
@@ -0,0 +1,72 @@
+//
+// Created by jordi on 07-05-20.
+//
+
+#ifndef RTT_FREEKICKTHEM_H
+#define RTT_FREEKICKTHEM_H
+
+#include <stp/Play.hpp>
+
+namespace rtt::ai::stp::play {
+
+class FreeKickThem : public Play {
+public:
+    /**
+     * Constructor that initializes roles with roles that are necessary for this play
+     */
+    FreeKickThem();
+
+    /**
+     * Gets the score for the current play
+     *
+     * On the contrary to isValidPlay() this checks how good the play actually is
+     * return in range of 0 - 100
+     *
+     * @param world World to get the score for (world_new::World::instance())
+     * @return The score, 0 - 100
+     */
+    uint8_t score(world_new::World* world) noexcept override;
+
+    /**
+     * Assigns robots to roles of this play
+     */
+    Dealer::FlagMap decideRoleFlags() const noexcept override;
+
+    /**
+     * Calculates info for the roles
+     */
+    void calculateInfoForRoles() noexcept override;
```

Comment:
Uninformative documentation. What kind of info does it calculate for the roles? When should you call this function? What is the reason for this design choice? What are the precondition when calling this function? How does the function work? All unanswered questions which some of them could be answered by adding more documentation.

Reply:
This function is pure virtual and documented in the base class